assets/js/vue/components/info-search/ResultSet.vue

```diff
@@ -49,14 +49,7 @@ export default {
   },
   computed: {
     resultCount() {
-      let resultCount = 0;
-      const informationProducts = this.results.informationProducts ?? [];
-      informationProducts.forEach((informationProduct) => {
-        if (informationProduct.published) {
-          resultCount += 1;
-        }
-      });
-      return resultCount;
+      return this.results.count;
```

Comment:
We can remove this, as it is not computing anything and we can display `this.results.count`